recipes/macos.rb

```diff
@@ -0,0 +1,114 @@
+# Install xquartz
+remote_file "/tmp/xquartz.pkg" do
+  source "https://github.com/XQuartz/XQuartz/releases/download/XQuartz-2.8.5/XQuartz-2.8.5.pkg"
+  not_if "pkgutil --pkg-info org.xquartz.X11"
+end
```

Comment:
@scpeters I ended up installing XQuartz via pkg download rather than homebrew cask because the homebrew cask approach does not work correctly when the user chef runs as isn't the `jenkins` user we want to be the homebrew owner.

Since this was a cask rather than a formula the main trade-off is that we have to manually update the XQuartz version in this recipe when we want to change the installed version (and the recipe only checks that the application is installed not that the version matches our desired).
However, since homebrew also does not support updating casks that have auto-update mechanisms I don't think this is a deal breaker.

Reply:
sounds good to me